recipe/build.sh

```diff
@@ -255,10 +246,17 @@ else
   _MAKE_TARGET=
 fi
 
+mkdir -p ${_buildd_shared}
+pushd ${_buildd_shared}
+  ${SRC_DIR}/configure "${_common_configure_args[@]}" \
+                       "${_dbg_opts[@]}" \
+                       --oldincludedir=${BUILD_PREFIX}/${HOST}/sysroot/usr/include \
+                       --enable-shared
+popd
+
 mkdir -p ${_buildd_static}
 pushd ${_buildd_static}
   ${SRC_DIR}/configure "${_common_configure_args[@]}" \
-                       "${_extra_opts[@]}" \
```

Comment:
This doesn't seem correct

Reply:
Which part? I deleted _extra_opts since it was only used to differentiate optimizations between the shared library and static lib. Totally open to other refactorings/suggestions

Reply:
Yeah, I guess I haven't had the time to review this well enough.

Reply:
Totally open to suggestions which would make this easier to review/merge. 

What do you suggest as next steps? Are there manual regression tests I can help run?